src/modules/contact-us/contact-us.service.ts

```diff
@@ -20,23 +22,25 @@ export class ContactUsService {
     try {
       await this.sendEmail(createContactDto);
     } catch (error) {
-      throw new InternalServerErrorException('Failed to send email');
+      console.log(error.message);
+      throw new InternalServerErrorException(SYS_MSG.INQUIRY_NOT_SENT);
```

Comment:
Don't throw internal server error. Let the global interceptor catch and log the error.

Reply:
noted.
Thank you for the review.
i have resolved this.